### DISCUSSION TITLE
A Nice Media Manager Should be there in filament.

### DISCUSSION THREAD

**Discussion by vikramparihar** [filamentphp/filament#789] [Ideas]
Well filament is very good tool for starting blog, article, and custom admin panel in Laravel.
But when i trying to move from 
https://backpackforlaravel.com/  "to" Filament , I found most important part like media manager should be there,
And also user can select any media from from input.
You can see the example here.
https://github.com/Laravel-Backpack/FileManager

![image](https://user-images.githubusercontent.com/5202442/145457489-c2a823a6-ddde-497d-b0fa-502a9e18c061.png)

And when user click on browse then a media manager should be open like below
![image](https://user-images.githubusercontent.com/5202442/145457745-3576ca42-d0f3-4dec-a21f-b3ba38f0935f.png)

Thank you.

**Comment by danharrin** [comment]
I can see someone selling a media plugin for Filament in the future, but this isn't something we're gonna add to the core as not everyone needs it.

**Reply by vikramparihar** [reply]
Thank you for response back.

**Reply by grahammtbr** [reply]
I might tackle a media/file manager in the new year. Learning Laravel/Livewire the past few months has greatly increased my desire to start moving clients away from WP. And, after testing Filament, I can see that's the direction I should take rather than creating something fully custom from scratch.

**Reply by aurawindsurfing** [reply]
@danharrin most of the users including me ask how to manage images in blog posts. It is not minority of users but majority imho. Thanks!

**Reply by danharrin** [reply]
https://filamentphp.com/plugins/media-library-pro

**Reply by aurawindsurfing** [reply]
FileUpload::make('attachments')->multiple() works like a charm, I did not realise at first that FileUpload was the same as ImageUpload field.

**Comment by halilcagriakkuzu** [comment]
you can use **[awcodes/filament-curator](https://github.com/awcodes/filament-curator)** package for this purpose. For those who needs

**Reply by aurawindsurfing** [reply]
Thanks will give it a go!

**Comment by dilab** [comment]
looking for something similar but a manger for files only.
